namespaces/live.cloud-platform.service.justice.gov.uk/specialist-provider-proto/resources/ecr.tf

```diff
@@ -0,0 +1,19 @@
+module "ecr-repo" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=7.0.0"
```

Comment:
change this line to:
source = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=7.1.0"